src/operator/tensor/elemwise_binary_op_extended.cc

```diff
@@ -26,9 +26,8 @@
 
 namespace mxnet {
 namespace op {
-MXNET_OPERATOR_REGISTER_BINARY(_power)
+MXNET_OPERATOR_REGISTER_BINARY_LAUNCH_CPU_DR(_power, mshadow_op::power)
```

Comment:
Actually, that will be good for _power_scalar, but that isn't yet implemented for sparse in this PR